src/js/controller/MainController.js

```diff
@@ -0,0 +1,96 @@
+import CarNameFormSection from '../view/CarNameFormSection.js';
+import TryCountFormSection from '../view/TryCountFormSection.js';
+import PlaySection from '../view/PlaySection.js';
+import WinnerSection from '../view/WinnerSection.js';
+import RacingcarModel from '../model/RacingcarModel.js';
+import { ALERT, GAME } from '../util/constants.js';
+import { moveCar } from '../util/service.js';
+
+export default class MainController {
+  constructor({
+    carNameFormSection,
+    tryCountFormSection,
+    playSection,
+    winnerSection
+  }) {
+    this.carNameFormSection = new CarNameFormSection(carNameFormSection);
+    this.tryCountFormSection = new TryCountFormSection(tryCountFormSection);
+    this.playSection = new PlaySection(playSection);
+    this.winnerSection = new WinnerSection(winnerSection);
+
+    this.racingCarModel = new RacingcarModel();
+
+    this.init();
+    this.bindEvents();
+  }
+
+  init() {
+    this.carNameFormSection.show().init();
+    this.tryCountFormSection.hide().init();
+    this.playSection.hide().render();
+    this.winnerSection.hide().render();
+  }
+
+  bindEvents() {
+    this.carNameFormSection.on('@submitCarNames', ({ detail }) =>
+      this.onSubmitCarNames(detail)
+    );
+
+    this.tryCountFormSection.on('@submitTryCount', ({ detail }) =>
+      this.onSubmitTryCount(detail)
+    );
+  }
+
+  onSubmitCarNames({ data }) {
+    const carNames = data.trim();
+
+    if (!this.isValidCarCount(carNames)) {
+      alert(ALERT.MIN_PEOPLE_LENGTH);
+      return;
+    }
+
+    if (!this.isValidCarName(carNames)) {
+      alert(ALERT.MAX_NAME_LENGTH);
+      return;
+    }
+
+    this.carNameFormSection.toggleDisableButton(true);
```

Comment:
toggleDisabledButton 이름이 toggle 시키는 함수명인데 상태를 넘기는게 어색하게 느껴지네요 

```suggestion
    this.carNameFormSection.disableButton();
```

Reply:
듣고 보니 그렇네요.. 네이밍 부분은 한번 더 고민해보겠습니다!